app/views/course/condition/_conditions.html.slim

```diff
@@ -0,0 +1,25 @@
+- conditional_name = conditional.class.name.demodulize.singularize.downcase
+
+h2
+  span = t('course.condition.header')
+
+div.dropdown.pull-right
+  button.btn.btn-primary.dropdown-toggle#new-condition-dropdown-btn [data-toggle="dropdown"
+    aria-haspopup="true"]
+    span = t('course.condition.new.header')
+  ul.dropdown-menu.dropdown-menu-right aria-labelledby="new-condition-dropdown-btn"
+    - Course::Condition::NAMES.each do |condition_name|
+      li= link_to t("course.condition.#{condition_name.pluralize}.new.header"),
+                  send("new_course_#{conditional_name}_condition_#{condition_name}_path",
```

Comment:
I read the source code. `path_for` is used for building path with param value and anchor like `abc?a=3&b=4#anchor`. I'm not sure how `path_for` is used here? like the following which isn't very helpful? 
```
path_for(path: '/course/achievements/1/condition/achievements/1'